pkg/internal/ebpf/tracer.go

```diff
@@ -95,7 +95,7 @@ func FindAndInstrument(ctx context.Context, cfg *ebpfcommon.TracerConfig, metric
 		return nil, fmt.Errorf("inspecting offsets: %w", err)
 	}
 
-	if goffsets != nil {
+	if goffsets != nil && !cfg.SkipGoSpecificTracers {
```

Comment:
maybe this condition should be used to conditionally execute this:
```go
	allFuncs := allGoFunctionNames(programs)
	elfInfo, goffsets, err := inspect(ctx, cfg, allFuncs)
```
So if you aren't going to instrument go specific tracers you don't need to previously search for Go function names nor offsets

Reply:
Yes, thank you!  That will avoid lots of unnecessary sifting through ELF files.  I have also realized I need to add documentation for the new option.

Reply:
Feedback implemented.  Thanks again!